third_party/terraform/tests/resource_storage_transfer_job_test.go

```diff
@@ -0,0 +1,185 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccStorageTransferJob_basic(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccStorageTransferJobDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccStorageTransferJob_basic(getTestProjectFromEnv()),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
This resource supports import, so we can use "import-style tests" instead of a `Check`: See https://github.com/terraform-providers/terraform-provider-google/blob/2.0.0/google/resource_compute_address_test.go#L23 for an example.

Reply:
Does this mean it would simply become:

```
func TestAccStorageTransferJob_basic(t *testing.T) {
	t.Parallel()

	resource.Test(t, resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
		Providers:    testAccProviders,
		CheckDestroy: testAccStorageTransferJobDestroy,
		Steps: []resource.TestStep{
			{
				ResourceName:      "google_storage_transfer_job.foobar",
				ImportState:       true,
				ImportStateVerify: true,
			},
		},
	})
}
```

Reply:
Not quite, but almost!

```
func TestAccStorageTransferJob_basic(t *testing.T) {
	t.Parallel()

	resource.Test(t, resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
		Providers:    testAccProviders,
		CheckDestroy: testAccStorageTransferJobDestroy,
		Steps: []resource.TestStep{
			{
				Config: testAccStorageTransferJob_basic(getTestProjectFromEnv()),
			},
			{
				ResourceName:      "google_storage_transfer_job. transfer_job",
				ImportState:       true,
				ImportStateVerify: true,
			},
		},
	})
}
```

Reply:
Done